src/fiatExchanges/SelectProvider.tsx

```diff
@@ -175,15 +231,17 @@ export default function SelectProviderScreen({ route, navigation }: Props) {
         flow={flow}
       />
       <LegacyMobileMoneySection
-        providers={asyncProviders.result?.legacyMobileMoneyProviders || []}
+        providers={legacyMobileMoneyProviders || []}
         digitalAsset={digitalAsset}
         flow={flow}
       />
-      <CoinbasePaymentSection
-        digitalAsset={digitalAsset}
-        cryptoAmount={route.params.amount.crypto}
-        coinbaseProvider={coinbaseProvider}
-      />
+      {coinbaseProvider && coinbasePayVisible ? (
```

Comment:
nit: Do you need to use a ternary here? [Can you just do](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator) `{(coinbaseProvider && coinbasePayVisible) && ...}`?

Reply:
Wait, `coinbaseProvider` is always truthy if `coinbasePayVisible` is true; can we just check that `coinbasePayVisible` is true instead?

Reply:
Even though <code>coinbasePayVisible</code> is dependent on <code>coinbaseProvider</code>, <code>coinbaseProvider={coinbaseProvider}</code> doesn't seem to recognize that coinbaseProvider must be defined unless I add coinbaseProvider in the ternary, or condition.

Reply:
The inline condition is helpful though. Didn't know that works

Reply:
Huh, are you saying Typescript complains that `coinbaseProvider` might be undefined? You could potentially use a nonnull assertion since we know it will be defined (i.e. `coinbaseProvider!`), but not strictly necessary (and not the best style -- strange that Typescript doesn't recognize that `coinbaseProvider` must be nonnull here...)

Reply:
At the moment, the condition is <code>coinbaseProvider && coinbasePayVisible && <..../></code>. Would this be more readable?